### ISSUE TITLE
Rogations, Ember days

### ISSUE THREAD

**Issue by igneus** [igneus/calendarium-romanum#57]
*Normae universales de anno liturgico et de calendario*, p. 45-47, leave the question of (re-)instituting rogation and ember days to conferences of bishops. Was this provision actually used anywhere? (In Czech Republic where I live it was not.) If yes, what (if any) is the status of these days in the liturgical calendar? Should support for them be added to calendarium-romanum?

**Comment by igneus** [comment]
[Notitia 85/1972, p. 273](http://www.cultodivino.va/content/dam/cultodivino/notitiae/1973/85.pdf#page=31) present a proposal of the Italian region (diocese/s?) Emilia-Flaminia, but of course it doesn't mean that the described solution is still in use today.
Given there's just an intention assigned to each day, it looks like the days didn't receive any special status in the liturgical calendar, just a "topic".

**Comment by igneus** [comment]
romcal/romcal#225 probably deals with the same matter (just approaching it from the other side).

**Comment by tukusejssirs** [comment]
@igneus, I don’t think that romcal/romcal#225 is a related issue. It deals with _days of prayer_, no with _ember days_, although ember days might be considered a subset of prayer days.

If I got it right, [_ember days_](https://www.farmersalmanac.com/ember-days) are [_kántrové dni_](http://www.trnava-tulipan.fara.sk/component/content/article/1-o-fare/217-kantrove-dni-kedy-su-preo-ich-treba-zachovava) in Slovak and indeed we celebrate them (although they are not obligatory).

I think the ember days should be implemented in both `calendarium-romanum` and `romcal` for particular calendars in which they are celebrated. Note that I think the dioceses / episcopal conferences might change their date.

**Comment by igneus** [comment]
I need to do some research on this matter, but I hold for very probable that "days of prayer" are a post-conciliar transformation (cf. GNLYC 46) of [rogations/rogation days](https://www.newadvent.org/cathen/13110b.htm). (Not of [ember days](https://www.newadvent.org/cathen/05399b.htm).)

If this assumption is correct, then my initial assertion that the provision of GNLYC 45-47 was not used in Czech Republic is wrong. We do have a list of *"prosebné dny"*, printed in the opening section of each year's ordo. These days, however, do not have any special liturgical features - just the day's entry in the ordo usually suggests that a particular intention should be inserted in the intercessions of the day's mass. Therefore this kind of "days of prayer" is completely out of `calendarium-romanum`'s scope (which is defined as "computing liturgical calendar in a narrow sense"). But if there are countries having them inscribed directly in the liturgical calendar like you wrote in romcal/romcal#225, that's definitely something worth attention.

**Comment by tukusejssirs** [comment]
I see what you mean.

Anyway, we (or at least me personally) try to make romcal a source of all informatation found in the _Direktórium_ (see romcal/romcal#20), which however takes some time to implement (we are a very small team).

> But if there are countries having them inscribed directly in the liturgical calendar like you wrote in romcal/romcal#225, that's definitely something worth attention.

Well, at least that is my impression from the sources I could find on the Internet.

**Comment by igneus** [comment]
**Austria:** the [directory 2019/2020 of dioceses Innsbruck and Feldkirch](https://www.dibk.at/content/download/98641/file/Direktorium_2020__web.pdf), p. 10f, states that Rogation days (celebrated on one or multiple days prior to the Ascension) should be observed in parishes where there is still a living tradition of doing so. Ember days (or rather "Ember weeks") are observed on the first week of Advent, first week of Lent, the week before Pentecost and the first week of October. Generally they are times of penance and prayer for vocations, particular topics for particular Ember days can be added by the bishops and are in such case listed in the directory.

**Comment by igneus** [comment]
See also the [dubium response](http://notitiae.ipsissima-verba.org/show/179) published in _Notitiae_ 5 (1969), p. 405.
